package.json

```diff
@@ -90,12 +88,12 @@
     "@babel/parser": "^7.8.3",
     "@babel/preset-env": "^7.8.3",
     "@babel/preset-typescript": "^7.8.3",
-    "@commitlint/cli": "^8.3.4",
+    "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.3.4",
     "@endpass/eslint-plugin-endpass": "^0.2.10",
     "@endpass/vue-i18n-translate-checker": "^0.0.6",
     "@semantic-release/changelog": "^3.0.6",
```

Comment:
I think, we do not need `semantic-release` absolutely, because we are making all releases manually